cypress/support/fragments/orders/orderLines.js

```diff
@@ -72,17 +73,49 @@ export default {
     }
   },
 
-  createPOLineViaActions: () => {
+  addPOLine: () => {
     cy.do([
       Accordion({ id: 'POListing' })
         .find(Button('Actions'))
         .click(),
       Button('Add PO line').click()
     ]);
   },
+
+  backToEditingOrder: () => {
+    cy.do(Button({ id: 'clickable-backToPO' }).click());
+  },
+
+  deleteOrderLine: () => {
+    cy.do([
+      PaneHeader({ id: 'paneHeaderorder-lines-details' }).find(actionsButton).click(),
+      Button('Delete').click(),
+      Button({ id: 'clickable-delete-line-confirmation-confirm' }).click()
+    ]);
+  },
+
+  POLineInfodorPhysicalMaterial: (orderLineTitleName) => {
+    cy.do([
+      TextField({ name: 'titleOrPackage' }).fillIn(orderLineTitleName),
+      Select({ name: 'orderFormat' }).choose('Physical resource'),
+      Button({ id: 'acquisition-method' }).click(),
+      SelectionOption('Depository').click(),
+      Select({ name: 'checkinItems' }).choose('Independent order and receipt quantity'),
+      physicalUnitPriceTextField.fillIn(physicalUnitPrice),
+      quantityPhysicalTextField.fillIn(quantityPhysical),
+      Select({ name: 'physical.materialType' }).choose('book'),
+      Button('Add location').click(),
+      Button('Location look-up').click(),
+      Select({ name: 'campusId' }).choose('Online'),
+      Button('Save and close').click(),
+      TextField({ name: 'locations[0].quantityPhysical' }).fillIn(quantityPhysical),
+      saveAndClose.click()
+    ]);
+  },
+
   fillInPOLineInfoViaUi: () => {
     cy.do([
-      TextField({ name: 'titleOrPackage' }).fillIn(`Autotest Tetle_${getRandomPostfix()}`),
+      TextField({ name: 'titleOrPackage' }).fillIn(orderLineTitle),
```

Comment:
may be good point to define specialOrderLineTitle was method parameter with orderLineTitle as default. And let's rename orderLineTitle to defaultOrderLineTitle

Reply:
Done